src/frontend/XAPI/XAPIStatement.ts

```diff
@@ -77,8 +80,42 @@ export const verbDefinition: {
 };
 
 export class XAPIStatement {
+  static toFixed(rawNumber: number, length: number = 3): number {
```

Comment:
imo this should probably not be on this class.

And `toFixed` is not the right name for it. Something to the effect of `truncateDecimalDigits` or something like that would be a clearer reflection of what you're doing. `toFixed` is known to return a string, not a number